README.md

```diff
@@ -9,17 +8,15 @@
 * VS Code (IDE)
 
 ## **Usage**
-To make use of this project run below below on any command prompt to have it on your local system.
+To make use of this project run below on any command prompt to have it on your local system.
 
 ```
-  git clone https://github.com/Ridwanullahi-code/github-flow.git
+  git clone https://github.com/Ridwanullahi-code/professional-portifolio
 ```
 or
 ```
-git clone git@github.com:Ridwanullahi-code/github-flow.git
+git clone git@github.com:Ridwanullahi-code/professional-portifolio.git
 ```
-
-
 ## **Show your support**
 
 Give a ⭐️ if you like this project!
```

Comment:
- [OPTIONAL] I would recommend adding the issue link in the contributing sections:

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

```